script.js

```diff
@@ -24,3 +25,107 @@ hamburger.addEventListener('click', toggleMenu);
 menuItems.forEach((menuItem) => {
   menuItem.addEventListener('click', toggleMenu);
 });
+
+const cardsData = [
+  {
+    name: 'Multi-Post Stories',
+    image: 'media/placholder.jpg',
+    technologies: [' CSS ', 'JavaScript', 'HTML'],
+    Source: 'https://github.com/Ferizad/Portfolio',
+    liveLink: 'https://github.com/Ferizad/Portfolio',
+  },
```

Comment:
- [ ] You have done excellently well with this approach :hugs: but considering the requirement, I think all your projects (6 in your case) should be stored as objects in an array, and there are minimum requirements for the key/value pair which I believe you are not adding up, to avoid hard-coding, especially the `project description`.
- Kindly add the other 5 project objects, with required properties :pray: 

Here's a snapshot from the part of the project requirement that can make it even clearer :recycle: 

![image](https://user-images.githubusercontent.com/99715640/224005202-4c63d355-3135-4643-a72f-80e64c072ea8.png)